OracleTuxedo/salt/samples/bankapp/start.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+cd /u01/data/bankapp/ || return 1
+
+APPDIR=$(pwd)
+export APPDIR
+
+UNAME1=$(uname -n)
+TUX_UID=$(id -u)
+TUX_GID=$(id -g)
+
+
+# clean up from any previous run
+tmshutdown -y &>/dev/null 
+rm -f TLOG GWTLOG tuxconfig saltconfig bankdl1 bankdl2 bankdl3 ULOG.*
+
+
+
+# modify and run the environment setup script 
+cp -p bankvar bankvar.new
+
+# shellcheck disable=SC2016
+echo '
+#
+# For GWWS
+#
+GWTLOGDEVICE=${APPDIR}/GWTLOG
+export GWTLOGDEVICE
+#
+# Device for binary file that gives SALT all its information
+#
+SALTCONFIG=${APPDIR}/saltconfig
+export SALTCONFIG
+' >> bankvar.new
```

Comment:
```suggestion
cat << 'EOF' > bankvar.new
#
# For GWWS
#
GWTLOGDEVICE="${APPDIR}/GWTLOG"
export GWTLOGDEVICE
#
# Device for binary file that gives SALT all its information
#
SALTCONFIG="${APPDIR}/saltconfig"
export SALTCONFIG
EOF
```

Reply:
Thanks. Appreciate it. Also, is there any benefit of using `cat` and HEREDOC over `echo` ?

Reply:
HEREDOC is designed for multi-line output, `echo` is not.

Reply:
okay, will change `echo` to HEREDOC soon.